lib/ruby_lsp/document.rb

```diff
@@ -177,6 +177,18 @@ def locate(node, char_position, node_types: [])
       [closest, parent, nesting.map { |n| n.constant_path.location.slice }]
     end
 
+    sig { returns(T::Boolean) }
+    def sigil_is_true_or_higher?
```

Comment:
I think it would be helpful to name this `sorbet_sigil_is_true_or_higher` since people reading this who aren't familiar with Sorbet may not know what it's referring to.